src/main/java/org/websync/supportedframeworks/jdi/CustomBaseTable.java

```diff
@@ -0,0 +1,18 @@
+package org.websync.supportedframeworks.jdi;
+
+import com.epam.jdi.light.elements.complex.table.BaseTable;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.XPath;
+import org.openqa.selenium.By;
+
+public class CustomBaseTable extends BaseTable {
```

Comment:
This is a valid case, but we also need more generic one - a class that inherits from the very base class for all JDI components. To find this base class - find ICoreElement interface and find class that implements this interface, if there are several such classes - please tell me and we will discuss this situation.